src/test/common/test_hobject.cc

```diff
@@ -201,7 +201,7 @@ namespace fmt {
 template <>
 struct formatter<test_hobject_fmt_t> {
 
-  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  auto parse(format_parse_context& ctx) { return ctx.begin(); }
```

Comment:
why is this change required?

Reply:
Because without this change I have compilation error
Which gcc version your environment has?

Reply:
I got this compilation error on Ubuntu and Fedora machines

Reply:
Was Fedora 37 (don't remember what the version was). I've just (last night) upgraded to 39 & gcc 13.2.1, and
will test there . On Clang - compiled OK on both platforms (and in the multiple  CI environments).

Anyway - can be merged now (I do not see a reason for a Teuthology test run for a single test change).

Reply:
Try to test on gcc11, because I think this is the default one for ceph